incident-disclosure: add Incident Disclosure and Notification policy (#12)
tailscale/security-policies: incident-response/index.md

* Modification or defacement of websites
* New open network ports on a system

> Tailscale regularly reviews logs for detecting and tracking attempted intrusions and other suspicious activity. These include git, cloud, networking, SaaS tool, and other infrastructure logs.

Reviewer: usage nit (up to you): "...reviews logs to detect and track..."

### Incident response and remediation

If a suspected incident is detected, it should be responded to following the [Incident response process](http://go/incident-response-process).

> We respond to reported incidents, and resolve and determine impact as soon as possible. We do not provide guarantees on time to remediate.

Reviewer 2: ```suggestion
We respond to reported incidents, and resolve and determine impact as soon as possible. We aim to remediate incidents as soon as possible.
```
